cli/ops/cephadm.py

```diff
@@ -142,7 +157,7 @@ def bootstrap(
     # Check for bootstrap configs
     for k in kw.keys():
         if isinstance(kw.get(k), dict):
-            kw[k] = generate_bootstrap_config(installer, kw.get(k))
+            kw[k] = generate_bootstrap_config(installer, nodes, kw.get(k))
```

Comment:
Add check for `apply-spec` key and call method to generate config.

Reply:
Updated